vtr_flow/tasks/regression_tests/vtr_reg_strong/strong_multiclock/config/config.txt

```diff
@@ -27,5 +24,5 @@ qor_parse_file=qor_multiclock.txt
 # Pass requirements
 pass_requirements_file=pass_requirements_multiclock.txt
 
-script_params=-starting_stage vpr
+script_params=-starting_stage vpr -sdc_file /tasks/regression_tests/vtr_reg_strong/strong_multiclock/config/multiclock.sdc
```

Comment:
Was it indented to add an SDC file here (since it wasn't specified before)?

Reply:
Yes, I tested the master branch and the sdc commands were applied.